### DISCUSSION TITLE
Exporting/Importing Language settings

### DISCUSSION THREAD

**Discussion by Xelalego** [mclemente/fvtt-module-polyglot#332] [Q&A] [answered]
How would one export the polyglot settings from one world and import them into another?

**Comment by mclemente** [comment, marked as answer]
On the console (F12):

1. Input `game.settings.get("polyglot", "Languages")`
2. Right-click the output and select "Copy Object"
3. On the world you want, input `game.settings.set("polyglot", "Languages", PASTE HERE)`

Do the same for `game.settings.get("polyglot", "Alphabets")` for fonts.
